src/plugins/intel_cpu/docs/debug_capabilities/README.md

```diff
@@ -24,3 +24,5 @@ Use the following cmake option to enable debug capabilities:
   `OV_CPU_SUMMARY_PERF=1`  
   Set the environment variable to display performance summary at the time when model is being destructed.  
   Internal performance counter will be enabled automatically. 
+* [Average counters](average_counters.md)  
```

Comment:
Maybe not mandatory but seems worthy to unite OV_CPU_SUMMARY_PERF and OV_CPU_AVERAGE_COUNTERS as single  variable with different output types: {filename.cvs, cout} (the same approach which is already adopted under DEBUG CAPS).

Reply:
The idea was actually to drop the ov_cpu_summary_perf
I think the combination of OV_CPU_AVERAGE_COUNTERS + aggregate-average-counters.py is basically a superset of OV_CPU_SUMMARY_PERF except the ability to output the result into cout. But do we really need the ability to print to cout?

Reply:
Agreed offline to unify OV_CPU_SUMMARY_PERF  and OV_CPU_AVERAGE_COUNTERS  debug capabilities in follow up PR.